frontend/ui/src/pages/Channels/MainPage/index.tsx

```diff
@@ -113,7 +114,7 @@ const SourcesInfo: SourceInfo[] = [
     image: <GoogleAvatarIcon />,
     newChannelRoute: '/',
     channelsListRoute: '/',
-    configKey: 'sources-google',
+    configKey: '',
```

Comment:
Maybe I'm not getting it, but how can this field be empty? 🤔

Reply:
yes I know it's counter-intuitive, but I had to remove Google's configKey in order to have the modal pop-up. 

the backend returns `sources-google: {enabled: true}` but we are not yet able to connect Google via the UI, so we want to show the modal box describing the necessary configuration to the user

Reply:
then you should add the `configKey` and the backend shoud return `enabled: false`. @AudreyKj  @chrismatix  @lucapette

Reply:
yeah @AudreyKj I'm afraid I misunderstod the problem here. We have an issue to fix this behaviour. But it won't work like @AitorAlgorta is mentioning. `enabled: false` will be returned when google is correctly configured _but_ for some reason is not running at the moment.

As of last week, the `/client.config` endpoint has a different contract. We return in the response only components that are present in the airy instance (which means they've been configured already). So the right condition for the flow of the channels new accounts pages is `source component is not null and enabled` to proceed with it.


Having said that: I would merge this anyway. And fix this back in #1599 (which I think is already being worked on by @paulodiniz)